components/vehiclespawner/fn_logiDoSpawnVehicle.sqf

```diff
@@ -46,18 +50,32 @@ private _spawnDir = getDir _logiVic; //teleport the spawned vic next to the spaw
 _spawnedVic setDir _spawnDir;
 _spawnedVic setVehiclePosition [_logiVic, [], 30, "NONE"];
 
+_spawnedVic spawn
+{
+    clearWeaponCargoGlobal _this;
+    clearMagazineCargoGlobal _this;
+    clearItemCargoGlobal _this;
+    clearBackpackCargoGlobal _this;
+};
```

Comment:
Any reason why you're spawning a new script for this, as opposed to directly running these commands on `_spawnedVic` within the main scope?

Reply:
From discord:
> when i run those commands i observe the process taking a while, the crates visibly drain out so i assume they take a while to run
rather than benchmarking them to see if they take seconds to run, i just threw em in a spawn so i don't have to deal with the timing

I did this with precisely 0 facts or logic so if you know it's not a problem I'll remove the spawn.